Automattic/wp-calypso
# Site Migration: Update Site Overwrite wording when confirming site overwrite

#### Changes proposed in this Pull Request

* ~~This PR improves the rendering of the confirm migration modal during Site Migration by reducing the line height on mobile screens.~~
* Updated the PR to only update the wording on the overwrite popup.

#### Testing instructions

* Use Calypso.live link
* Go to a WordPress.com Business site 
* Go to Tools -> Import -> WordPress
* Enter a Jetpack enabled site address (of a site you own) 
* Continue until you reach the screen where you need to confirm the start of the migration and that it will overwrite the site.
* Check if the text appears and reads well.
